integration/airflow/tests/integration/docker-compose.yml

```diff
@@ -1,4 +1,36 @@
 version: "3.7"
+
+x-airflow-common: &airflow-common
+  build:
+    context: .
+    target: airflow
+  environment: &airflow-common-env
```

Comment:
Do you need to set `&airflow-common-env`?

Reply:
Removed.